.github/workflows/build.yml

```diff
@@ -0,0 +1,28 @@
+name: Build
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
```

Comment:
This could be used as an opportunity to add support for JDK 17.

Reply:
I would rather have Java 17 support as a separate PR as this one is big as it is already.